website/content/docs/runner/on-demand-runner.mdx

```diff
@@ -0,0 +1,168 @@
+---
+layout: docs
+page_title: On-Demand Runners
+description: |-
+  A brief overview about on-demand runners and their role in Waypoint.
+---
+
+# On-Demand Runners
+
+On-demand runners allow a single Waypoint installation to scale very high by
+leveraging the defined platform to spawn resources to execute Waypoint jobs
+remotely. Through on-demand runner profiles and runner targeting, Waypoint can
+even utilize resources from other platforms that Waypoint has access to. For
+example, a local Kubernetes server install could be configured to use Amazon ECS
+to execute jobs.
+
+## Job Assignment
+
+When a job is submitted to the server, if on-demand runners are enabled, it
+will spawn an on-demand runner and the job is placed in a holding set. This is
+the same mechanism we use today for automatic CLI runners. When the on-demand
+runner starts, it will execute the normal Waypoint runner code and connect back
+to the server. On connection, the server will see the job in the holding set
+and send it immediately to the new runner.
+
+The flow of information about the job will use the existing grpc runner APIs so
+all the input and output remain the same as today.
```

Comment:
Similar to my comment above, “as today” is good framing for release notes but less so for “timeless” docs. Perhaps contrast it with the CLI-runner again?

```suggestion
The flow of information about the job uses the normal grpc runner APIs so all
the input and output remain the same as if the work were performed by the
CLI-runner.
```